### PR TITLE
report error on breakpoint with invalid line num

### PR DESCRIPTION
**Pull request by ofrobots**
@matthewloring PTAL.

### CODE REVIEW

File: lib/v8debugapi.js [hunk 1]
```diff
@@ -277,6 +280,12 @@ module.exports.create = function(logger_, config_) {
     // TODO: need test coverage for the above
     // TODO: work with debug server to come up with a more robust solution
 
+    if (breakpoint.location.line >= matchingScript.line_ends.length) {
+      return setErrorStatusAndCallback(cb, breakpoint,
+        StatusMessage.BREAKPOINT_SOURCE_LOCATION,
+        messages.INVALID_LINE_NUMBER);
```
Comment (matthewloring) [inline lib/v8debugapi.js]: Nit: indentation

**Comment by matthewloring** [comment]
LGTM after the nit.

File: lib/v8debugapi.js (hunk 1 above)
Comment (ofrobots) [inline lib/v8debugapi.js]: It is consistent with the other occurrences of setErrorStatusAndCallback in the file. Also, we need to pick a style-guide to refer to for this; and somehow enforce as well. What style guide should we use?

Comment (matthewloring) [inline lib/v8debugapi.js]: Ahh, I see that now. [The Google one](https://google.github.io/styleguide/javascriptguide.xml) seems like a good place to start but I am not yet familiar with it.

Comment (matthewloring) [inline lib/v8debugapi.js]: This patch appears to just check for scripts loaded by v8. We will have to make sure it applies generally when we move away from using v8 to find scripts.

Comment (ofrobots) [inline lib/v8debugapi.js]: Yes. The test must continue to be functional across any changes to how we figure out the the list of scripts. That's why this test needs to be added before we do any large refactoring of the script validation code.

Comment (ofrobots) [inline lib/v8debugapi.js]: The [style guide](https://google.github.io/styleguide/javascriptguide.xml?showone=Code_formatting#Code_formatting) seems the allow the indentation I used, with the caveat that it prefers indent = 4 spaces while our code is indent = 2.

Comment (matthewloring) [inline lib/v8debugapi.js]: Yup, it looks weird to me on the github diff but it's not in violation of the guide.

**Comment by matthewloring** [comment]
LGTM.

**Comment by ofrobots** [comment]
Landed in https://github.com/GoogleCloudPlatform/cloud-debug-nodejs/commit/fbac50d6570a1eb43ed0b72a9f4676e8d09a783f.

**Pull request closed without merging**